core/storage/question/gae_models.py

```diff
@@ -246,6 +246,23 @@ def get_models_by_skill_id(cls, skill_id):
         return QuestionSkillLinkModel.query().filter(
             cls.skill_id == skill_id).fetch()
 
+    @classmethod
+    def get_models_by_question_id(cls, question_id):
+        """Returns a list of QuestionSkillLink domains of a particular
+        question ID.
+
+        Args:
+            question_id: str. ID of the question.
+
+        Returns:
+            list(QuestionSkillLinkModel)|None. The list of question skill link
```

Comment:
Ditto -- question skill link domains --> question skill link models.

Domain objects refer to stuff defined in core/domain ("plain old python objects").

Reply:
fixed